src/Taxonomies/Term.php

```diff
@@ -38,7 +38,7 @@ public static function fromModel(Model $model)
         $term->data($data);
 
         if (config('statamic.system.track_last_update')) {
-            $term->set('updated_at', $model->updated_at ?? $model->created_at);
+            $term->set('updated_at', ($model->updated_at ?? $model->created_at)->timestamp);
```

Comment:
Nice, only issue is that a future retrieval of a term, e.g. `TermFacade::query()->first()->updated_at` won't work as it's already an integer

Not sure if this is the most elegant way of solving it:
```suggestion
            $updatedAt = $model->updated_at ?? $model->created_at;

            $term->set('updated_at', $updatedAt instanceof Carbon ? $updatedAt->timestamp : $updatedAt);
```

Reply:
No this is only called when a term is pulled from the database and made into a statamic term.

Reply:
I could be doing something wrong then

I added `$this->assertEquals(now(), TermFacade::query()->first()->updated_at);` to the test and got an error. I didn't look at it for long though, I assumed something was cached. Sorry if I'm leading you on a wild goose chase!

Reply:
Hmm ok not quite sure what's going on, something is being changed that shouldn't be in the LocalizedTerms. I'll apply your fix but I want to revisit it as somethings not right.